test/integration/backward_compatible/sql/DataTypeTest.js

```diff
@@ -73,11 +74,54 @@ describe('Data type test', function () {
     };
 
     before(async function () {
+        serverVersionNewerThanFive = await TestUtil.compareServerVersionWithRC(RC, '5.0') >= 0;
+        const CLUSTER_CONFIG = serverVersionNewerThanFive ? JET_ENABLED_CONFIG : null;
+
         TestUtil.markClientVersionAtLeast(this, '4.2');
-        cluster = await RC.createCluster(null, JET_ENABLED_CONFIG);
+        cluster = await RC.createCluster(null, CLUSTER_CONFIG);
         await RC.startMember(cluster.id);
     });
 
+    /**
+     * Creates portable mapping for SQL queries. In 5.0, users started to write explicit mapping for SQL queries against maps.
+     * @param keyFormat Key format
+     * @param factoryId Portable's factory id
+     * @param classId Portable's class id
+     * @param columns Columns as a dict where keys are column names, and values are case insensitive value formats.
+     */
+    const createMappingForPortable = async (keyFormat, factoryId, classId, columns) => {
+        if (!serverVersionNewerThanFive) {
+            // Before 5.0, mappings are created implicitly, thus we don't need to create explicitly.
+            return;
+        }
+
+        const entries = Object.entries(columns);
+
+        const numberOfColumns = entries.length;
+
+        const columnsString = entries.reduce((accumulator, current, currentIndex) => {
+            return accumulator + `${current[0]} ${current[1].toUpperCase()}${currentIndex !== numberOfColumns - 1 ? ',' : ''}\n`;
+        }, '');
```

Comment:
Would something like this be more readable?

```js
Object.entries(columns).map(column => `${column[0]} ${column[1].toUpperCase()}`).join(',\n');
```

Reply:
yeah thanks